docs/jade-getting-started.md

```diff
@@ -191,10 +188,11 @@ gcloud container clusters get-credentials dev-master --region us-central1 --proj
 ```
 
 ## 8. Configure Azure
+In order to do Azure development, do the following steps:
```

Comment:
I thought about putting an "optional" here, given that we aren't doing that much azure work these days, but thought this was fine.

Reply:
I think that's a good idea, we've talked internally about approaching Azure in a totally different way, so this might not be relevant.